app/Http/Requests/CreateUserRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:255',
+            'email' => 'required|email|unique:users'
+        ];
+    }
+    public function messages()
+    {
+        return [
+            'name.max' => 'Số kí tự phải nhỏ hơn 255 ký tự',
+            'name.required' => 'Name không được để trống',
+            'email.required' => 'Email không được để trống',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma